src/stars.css

```
@@ -0,0 +1,41 @@
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+/* .stars + .stars {
+  margin-top: 4px;
+} */
+
+.stars--1 :nth-child(-n+1) {
```

```suggestion
.stars--1 :nth-child(-n+1),
stars--2 :nth-child(-n+2),
.... {
```
Consider grouping selectors with commas